Editor/SettingsAnalysis/PlayerSettingsAnalyzer.cs

```diff
@@ -44,11 +44,42 @@ class PlayerSettingsAnalyzer : ISettingsModuleAnalyzer
             })
         };
 
+        static readonly Descriptor k_IL2CPPCompilerConfigurationMasterDescriptor = new Descriptor(
+            "PAS1004",
```

Comment:
I changed how we declare IDs. Let's use [this ](https://github.com/Unity-Technologies/ProjectAuditor/blob/3a8465ee4b4ca454932a354daae62a87eabff985/Editor/SettingsAnalysis/GraphicsApiAnalyzer.cs#L17) pattern. By doing that, we can replace all hardcode string IDs accordingly.